ajax/libs/twitter-fetcher/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "twitter-fetcher",
+  "version": "16.0.2",
+  "filename": "js/twitterFetcher_min.js",
+  "keywords": [
+    "Twitter",
+    "OAUTH",
+    "JavaScript",
+    "tweets",
+    "api",
+    "widget"
+  ],
+  "description": "Fetch your twitter posts without using the new Twitter 1.1 API. Pure JavaScript!",
+  "author": "Jason Mayes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jasonmayes/Twitter-Post-Fetcher.git"
+  },
+  "homepage": "http://www.jasonmayes.com/projects/twitterApi/#sthash.PwyUJPV4.wJnsNgKd.dpbs",
+  "license": "MIT",
+  "npmName": "twitter-fetcher",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/style*",
+        "js/+(twitterFetcher*|exampleUsage*)"
```

Comment:
https://github.com/jasonmayes/Twitter-Post-Fetcher/blob/master/index.html#L35
like style.css is used in index.html for an example usage, I think exampleUsage.js has the same usage.